dashboard/DashboardDB.py

```diff
@@ -334,6 +367,27 @@ def get_user_chours(self, hpc: const.HPC, username: str):
 
         return [UserChEntry(*result) for result in results]
 
+    def check_update_time(self, last_update_time_string, current_update_time):
+        """
+        Checks whether the time gap between update times exceeds the idling time limit
+        if exceeds, regards as a collection error.
+        :param last_update_time_string:
```

Comment:
Specifying the parameters without a proper description isn't overly useful, so maybe remove these and add use type hinting if you want to specify the types?

Reply:
updated